### PR TITLE
Create class Animated Entity

### PR DESCRIPTION
**Pull request by Orrigine**
Animated Entity inherit from Entity, with fonctions that animate the sprite

### CODE REVIEW

File: headers/graphics/AnimatedEntity.hpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	/// <summary>
+	/// Get the current frame number.
+	/// </summary>
+	int getCurrentFrame();
+
+	/// <summary>
+	/// Set the current frame.
+	/// </summary>
+	/// <param name="frame">Frame number</param>
+	void setCurrentFrame(int);
```
Comment (nwiart) [inline headers/graphics/AnimatedEntity.hpp:31]: Name the parameters, if you're going to give them a name in the documentation above

File: source/graphics/AnimatedEntity.cpp [hunk 2]
```diff
@@ -0,0 +1,69 @@
+#include "../../headers/graphics/AnimatedEntity.hpp"
```
Comment (nwiart) [inline source/graphics/AnimatedEntity.cpp:1]: Include from root please...

File: source/graphics/Entity.cpp [hunk 3]
```diff
@@ -1,9 +1,9 @@
-#include "headers/graphics/Entity.hpp"
+#include "../../headers/graphics/Entity.hpp"
```
Comment (nwiart) [inline source/graphics/Entity.cpp:1]: Same here

File: source/graphics/AnimatedEntity.cpp (hunk 2 above)
Comment (nwiart) [inline source/graphics/AnimatedEntity.cpp:1]: Include from root please...
(add `-I./` to GCC command line)

```suggestion
#include "headers/graphics/AnimatedEntity.hpp"
```

**Review by Wazy42 — APPROVED** [review]

**Pull request merged**